terms/experimentalData/assay.json

```diff
@@ -438,6 +438,11 @@
             "description": "SNP measurements captured by array technology",
             "source": "http://purl.obolibrary.org/obo/OBI_0001204"
         },
+        {
+            "const": "snrnaSeq",
```

Comment:
scrnaSeq and snrnaSeq look very similar. 😆

Reply:
Sure do! Fingers crossed this reduces confusion instead of increasing it... 😬